LoadInterests.py

```diff
@@ -0,0 +1,13 @@
+import csv
+
+def main(interests):
+    # code to use the interests in the main function
+    pass
+
+with open('interests.csv', 'r') as file:
+    csv_reader = csv.reader(file)
+    interests = []
+    for row in csv_reader:
+        interests.append(row[0])
+    only_interests = interests_with_header[1:]
+    main(interests)
```

Comment:
the name of the file should be load_interests.py, it should go under `src/data_loaders` dir

Reply:
done